src/Sylius/Bundle/ShopBundle/Resources/views/Product/_info.html.twig

```diff
@@ -1,7 +1,11 @@
 {% set product = variant.product %}
 
 <div class="ui header">
-    {% include '@SyliusShop/Product/_mainImage.html.twig' with {'product': product, 'filter': 'sylius_shop_product_tiny_thumbnail'} %}
+    {% if variant.hasImages %}
+        {% include '@SyliusShop/Product/_mainImage.html.twig' with {'product': variant, 'filter': 'sylius_shop_product_tiny_thumbnail'} %}
```

Comment:
This `product` variable is not the best naming since we are passing `variants` as well :( Could we make some unification?